### PR TITLE
New icons ready to go

### PR DESCRIPTION
**Pull request by leoheck**
@Foggalong, take a look all looks good now.

### CODE REVIEW

**Comment by Foggalong** [comment]
Merge conflict here. Think it's the INex line because that's already in

**Comment by leoheck** [comment]
I just copy the master tofix.csv to add my modifications (again). But I don't understand why it cannot be automatically merged. Any clue?

**Comment by leoheck** [comment]
Awsome \o/!!!!

File: tofix.csv [hunk 1]
```diff
@@ -218,8 +220,11 @@ qPDFview,qpdfview,/usr/share/qpdfview/qpdfview.svg,qpdfview
 Qucs,qucs,/usr/share/pixmaps/big.qucs.xpm,qucs
 R,r,/usr/share/pixmaps/r.png,rlogo_icon
 Raccoon,raccoon,/usr/share/pixmaps/raccoon.svg,raccoon
+Reditr,chrome-pmfcbbijgnhoebddbjpmlikabnbnddgb-Default ,reditr,reditr
```
Comment (Foggalong) [inline tofix.csv]: This isn't hardcoded it's, just the standard format for icon names with Chrome extensions

File: tofix.csv [hunk 2]
```diff
@@ -242,6 +252,10 @@ SportsTracker,SportsTracker,/opt/SportsTracker/SportsTracker.png,sportstracker
 Springseed,springseed,/usr/share/pixmaps/springseed/springseed.svg,springseed
 SQLiteBrowser,sqlitebrowser,/usr/share/pixmaps/sqlitebrowser.svg,sqlitebrowser
 SQuirrel SQL,squirrel-sql,/usr/share/pixmaps/acorn.ico,squirrel-sql
+<<<<<<< HEAD
+SSH Tunnel Manager ,gstm,gSTM.png,lock
+=======
+>>>>>>> upstream/master
```
Comment (Foggalong) [inline tofix.csv]: What's this about?

File: tofix.csv [hunk 3]
```diff
@@ -273,8 +287,10 @@ Variety,variety,/opt/extras.ubuntu.com/variety/share/variety/media/variety.svg,v
 Variety,extras-variety,/opt/extras.ubuntu.com/variety/share/variety/media/variety.svg,variety
 Viber,viber,/usr/share/pixmaps/viber.png,viber
 Visual Studio,visual-studio-code,/home/${SUDO_USER:-$USER}/tools/web/visual-studio-code/resources/app/vso.png,visual-studio-code
+Viva,viva,viva.png,viva
 Warmux,warmux,warmux_128x128.png,warmux
 Widelands,widelands,/usr/share/games/widelands/pics/wl-ico-64.png,widelands
+WinUSB,winusbgui,winusbgui-icon.png,web-microsoft
```
Comment (Foggalong) [inline tofix.csv]: This isn't the right icon name. Replace `web-microsoft` with `winusb`

**Comment by Foggalong** [comment]
@leoheck there are a few small problem with the pull that I'd not noticed until just now. sorry!

**Comment by leoheck** [comment]
Yes, sorry for that. I have updated the mistakes.

**Comment by Foggalong** [comment]
Thanks! :D

**Comment by leoheck** [comment]
:+1:  :)

**Pull request merged**